vosekast_control/TestSequence.py

```diff
@@ -124,9 +121,8 @@ def __init__(
                     )
                 if self.emulate:
                     # timeout
-                    time_sequence_t1 = datetime.now()
-                    time_sequence_passed = time_sequence_t1 - time_sequence_t0
-                    delta_time_sequence = time_sequence_passed.total_seconds()
+                    time_sequence_t1 = time.time() * 1000
+                    delta_time_sequence = (time_sequence_t1 - time_sequence_t0) / 1000
```

Comment:
you store that time not in database. you only check if a time different is bigger then 30s. 
so the change to ms make no sense here and add unnecessary multiplication and devisions.

Reply:
makes sense